### PR TITLE
Adjust the deferred scripts

### PR DESCRIPTION
**Pull request by dgrammatiko**
- Scripts with `type=module` are deferred by default [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#other_differences_between_modules_and_standard_scripts) thus they should be in the same level as those with the `defer` attribute.
- Also updated the sync level css selector to ignore the same rule ([src][[type*=module]])
- This PR patches both scripts (+ webpagetest)

### CODE REVIEW

**Review by rviscomi — COMMENTED** [review]
Sorry! I had this comment sitting in Pending this whole time.

Do we need `*=`?

File: capo.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ function isImportStyles(element) {
 }
 
 function isSyncScript(element) {
-  return element.matches('script:not([src][defer],[src][async],[type*=json])')
+  return element.matches('script:not([src][defer],[src][type*=module],[src][async],[type*=json])');
```
[the remark this answers is not in the document]
Comment (dgrammatiko) [reply capo.js]: Probably a liberal copy/paste

File: capo.js [hunk 2]
```diff
@@ -87,7 +87,7 @@ function isPreload(element) {
 }
 
 function isDeferScript(element) {
-  return element.matches('script[src][defer]');
+  return element.matches('script[src][defer]') || element.matches('script:not([src][async])[src][type*=module]');
```
Comment (rviscomi) [inline capo.js:90]:
```suggestion
  return element.matches('script[src][defer]') || element.matches('script:not([src][async])[src][type=module]');
```

File: webpagetest/capo.js [hunk 3]
```diff
@@ -69,7 +69,7 @@ function isImportStyles(element) {
 }
 
 function isSyncScript(element) {
-  return element.matches('script:not([src][defer],[src][async],[type*=json])')
+  return element.matches('script:not([src][defer],[src][type*=module],[src][async],[type*=json])');
```
Comment (rviscomi) [inline webpagetest/capo.js:72]:
```suggestion
  return element.matches('script:not([src][defer],[src][type=module],[src][async],[type*=json])');
```

File: webpagetest/capo.js [hunk 4]
```diff
@@ -81,7 +81,7 @@ function isPreload(element) {
 }
 
 function isDeferScript(element) {
-  return element.matches('script[src][defer]');
+  return element.matches('script[src][defer]') || element.matches('script:not([src][async])[src][type*=module]');
```
Comment (rviscomi) [inline webpagetest/capo.js:84]:
```suggestion
  return element.matches('script[src][defer], script:not([src][async])[src][type=module]');
```

File: capo.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (rviscomi) [reply capo.js]: Ok cool I'll merge edits for the other instances

**Pull request merged**